Add openssl requirement to contributing guidelines (#90)
modrinth/docs: docs/details/contributing.md

sqlx database setup
```

> Finally, if on Linux, you will need the OpenSSL library, which can be installed with the following command (command may vary depending on package manager):
> 
> ```bash
> sudo apt update
> sudo apt install pkg-config libssl-dev
> ```

Reviewer: ```suggestion
Finally, if on Linux, you will need the OpenSSL library. On Debian-based systems, this involves the `pkg-config` and `libssl-dev` packages.
```
